src/blocks/stars.css

```
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+}
+
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
```

Don't need to duplicate styles
```suggestion
.stars__star {
  background-image: url(../images/star.svg);
  background-repeat: no-repeat;
  background-position: center;
  height: 16px;
  width: 16px;
}

.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
  background-image: url(../images/star-active.svg);
}
```